dubbo-common/src/main/java/org/apache/dubbo/config/AbstractConfig.java

```diff
@@ -559,6 +560,13 @@ public void refresh() {
         postProcessRefresh();
     }
 
+    private void invokeSetParameters(Map<String, String> values) {
+        Map<String, String> map = invokeGetParameters(getClass(), this);
```

Comment:
It is better to check argument `Map<String, String> values`, If it is null or empty, just return

Reply:
ok, i will modify it.